docs/tasks/archive/2026/08/20260822-release-v0.6.5-lessons.md

```
@@ -0,0 +1,76 @@
+was genuinely green. What catches it is reading the live state:
+
+```bash
+kubectl -n wafflebase get pods -o custom-columns=\
+'NAME:.metadata.name,AGE:.metadata.creationTimestamp,IMAGE:.spec.containers[0].image'
+```
+
+A pod `creationTimestamp` older than the release means the release is not
+deployed, regardless of what ArgoCD or the image tag says.
+
+For v0.6.6: either pin every release (accepting mid-cycle `:latest` PRs like
+#326/#339 when an unreleased feature is needed), or make the rollout an
```

markdownlint-cli2 (0.23.2)

[warning] No space after hash on atx style heading

(MD018, no-missing-space-atx)